src/test/java/reposense/commits/CommitInfoAnalyzerTest.java

```diff
@@ -18,6 +18,7 @@
 
     @Test
     public void analyzeCommits_allAuthorNoIgnoredCommitsNoDateRange_success() {
+        config.getAuthorAliasMap().clear();
```

Comment:
Can elaborate why it is `clear()` is needed now?

Reply:
Is it better if we clear the state of `AuthorAliasMap` in a `@Before` method?

Reply:
I am considering removing the redundant adding of alias like `config.getAuthorAliasMap().put(EUGENE_AUTHOR_NAME, new Author(EUGENE_AUTHOR_NAME));` in the following method but keep the `clear()` only at where it is needed.

Reply:
Sorry for not looking on `GitTestTemplate`. Since it uses an `AllAliasAuthor` in config at the beginning, so maybe we can just put the `clear()`   in a @Before method, I will edit it and push it.

Reply:
Deleting aliases is harder for code maintenance as the code author would need to know which aliases to delete.

The problem you faced was caused by the use of same method name, so it overrides parent method. 
So, you can do this 😃 

```
@Before
public void before() throws InvalidLocationException {
    super.before();
    config.getAuthorAliasMap().clear();
}
```

@yamidark
We may need to review and improve `GitTestTemplate`.